Assets/MixedRealityToolkit.Extensions/HandPhysicsService/README.md

```diff
@@ -0,0 +1,17 @@
+# Hand Physics Extension Service
```

Comment:
Is there an entry in the toc.yml file? If not, please add one.

Reply:
Added an entry and detail page to docs. 👍